src/MiniExcel/DbList.cs

```diff
@@ -0,0 +1,176 @@
+        }
+
+        public void Add(string item)
+        {
+            var maxIndex = GetMaxIndex();
+            _cmd.CommandText = $"INSERT INTO {_tableName}(name, `index`) VALUES ('{item}', {maxIndex + 1})";
+            _cmd.ExecuteNonQuery();
+            Count += 1;
+        }
+
+        private long GetMaxIndex()
+        {
+            _cmd.CommandText = $"SELECT MAX(`index`) FROM {_tableName}";
+            var result = _cmd.ExecuteScalar();
+            if (result == DBNull.Value)
+                return -1;
+
+            return (long)result;
+        }
+
+        public void Clear()
+        {
+            _cmd.CommandText = $"DELETE FROM {_tableName}";
+            _cmd.ExecuteNonQuery();
+            Count = 0;
+        }
+
+        public bool Contains(string item)
+        {
+            _cmd.CommandText = $"SELECT * FROM {_tableName} WHERE name = '{item}'";
+            return _cmd.ExecuteScalar() != null;
+        }
+
+        public void CopyTo(string[] array, int arrayIndex)
+        {
+            throw new System.NotImplementedException();
+        }
+
+        public void AddRange(List<string> array)
+        {
+            var maxIndex = GetMaxIndex();
+
+            var cmdTxt = new StringBuilder();
+
+            cmdTxt.Append($"INSERT INTO {_tableName}(name, `index`) VALUES");
+            for (var i = 0; i < array.Count; i++)
+            {
+                var item = array[i];
+                cmdTxt.Append($"('{item}', {maxIndex + i + 1})");
+                cmdTxt.Append(i != array.Count - 1 ? ',' : ';');
+            }
+
+            _cmd.CommandText = cmdTxt.ToString();
+            _cmd.ExecuteNonQuery();
+            Count += array.Count;
+        }
+
+        public bool Remove(string item)
+        {
+            var index = IndexOf(item);
+            RemoveAt(index);
+            return true;
+        }
+
+        public int Count { get; private set; }
+
+        public bool IsReadOnly { get; }
+
+        public int IndexOf(string item)
+        {
+            _cmd.CommandText = $"SELECT `index` FROM {_tableName} WHERE name = '{item}'";
+            return Convert.ToInt32(_cmd.ExecuteScalar());
+        }
+
+        public void Insert(int index, string item)
+        {
+            _cmd.CommandText = $"UPDATE {_tableName} SET `index` = `index` + 1 WHERE `index` >= {index}";
+            _cmd.ExecuteNonQuery();
+            _cmd.CommandText = $"INSERT INTO {_tableName}(name, `index`) VALUES ('{item}', {index})";
+            _cmd.ExecuteNonQuery();
+            Count += 1;
+        }
+
+        public void RemoveAt(int index)
+        {
+            _cmd.CommandText = $"DELETE FROM {_tableName} WHERE `index` = {index}";
+            _cmd.ExecuteNonQuery();
+            _cmd.CommandText = $"UPDATE {_tableName} SET `index` = `index` - 1 WHERE `index` > {index}";
+            _cmd.ExecuteNonQuery();
+            Count -= 1;
+        }
+
+        public string this[int index]

```

Comment:
I think it's not a stable. you should switch the config and set cache size. Default size is 5MB

MiniExcel.Query(path, configuration: new OpenXmlConfiguration() { EnableSharedStringCache = true }).First();

Reply:
My point is the code stores data but seems like didn't get any data from SQLite DB, and the `Data Source` will be random from your construct method, that source might not be reuse went you run again right?

```
var dbList = new DbList(Guid.NewGuid().ToString());
```

Reply:
Previous` sharestrings` implementation is `List<string>`, I replace `List<String>` with `IList<String>` and create `DbList` to inherit. 
Get `sharedstrings` by index, that's why index in DbList will query sqlite
https://github.com/MiniExcel/MiniExcel/blob/bbb060147964b5bf3fc14abd0db74d2f27364fe9/src/MiniExcel/OpenXml/ExcelOpenXmlSheetReader.cs#L687

Reply:
Hi @Weilence  OK I see your point, Thank you for your description 

I find there two issues might be notice memory leak & file didn't delete problem. because `ExcelOpenXmlSheetReader` class didn't call `DBList.Dispose` method which might release unmanaged resources currently.

There are a lot of extension files that are `.db` exists on the bin path right now.

<img width="348" alt="image" src="https://user-images.githubusercontent.com/9159452/160241535-01f0d177-1feb-4733-aaa6-bd7a30f1738a.png">